webapp/src/Controller/Jury/ContestController.php

```diff
@@ -504,8 +508,61 @@ public function editAction(Request $request, int $contestId)
                 $problem->setContest($contest);
             }
 
+            // Determine the removed teams, team categories and problems.
+            // Note that we do not send out create / update events for
+            // existing / new problems, teams and team categories. This happens
+            // when someone connects to the event feed (or we have a
+            // dependent event) anyway and adding the code here would
+            // overcomplicate this function
+            $deletedEntities = function(Collection $collection, string $idMethod) {
+                /** @var PersistentCollection $collection */
+                $deletedEntities = [];
```

Comment:
This `$deletedEntities` is different from the variable that the function is assigned to, right? I think for clarity it's better to give them separate names. Also, if I understand correctly, then `getSnaphost()` returns the data that's currenty in the DB, right? Might be helpful to explain that in the comments at the top.